lib/jekyll/assets.rb

```diff
@@ -13,7 +13,7 @@
 ]
 
 requires.each do |part|
-  Pathutil.new(File.expand_path("../", __FILE__)).join("assets", part.to_s)
+  Pathutil.new(File.expand_path(__dir__)).join("assets", part.to_s)
```

Comment:
There's no need in `expand_path`. Use `__dir__` directly:

``` ruby
Pathutil.new(__dir__).join("assets", part.to_s)
```

Reply:
fixed.